packages/paste-core/components/form/stories/index.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import {useUID} from 'react-uid';
+import {storiesOf} from '@storybook/react';
+import {withKnobs, boolean, text, select} from '@storybook/addon-knobs';
+import {Box} from '@twilio-paste/box';
+import {Flex} from '@twilio-paste/flex';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+import {FormInput, FormLabel, FormHelperText, InputTypes} from '../src';
+
+const inputTypeOptions = ['text', 'email', 'hidden', 'number', 'password', 'search', 'tel'];
+
+storiesOf('Components|Form', module)
+  .addDecorator(withKnobs)
+  .add('Input Options', () => {
+    const hasError = boolean('hasError', false);
+    const isDisabled = boolean('disabled', false);
+    const isReadOnly = boolean('readOnly', false);
+    const isRequired = boolean('required', false);
+    const inputTypeValue = select('type', inputTypeOptions, 'text') as InputTypes;
+    return (
+      <>
+        <FormLabel for={text('for', 'input_field')} disabled={isDisabled} required={isRequired}>
+          Label
+        </FormLabel>
+        <FormInput
+          disabled={isDisabled}
+          hasError={hasError}
+          id={text('id', 'input_field')}
+          name={text('name', '')}
+          placeholder={text('placeholder', 'Placeholder')}
+          readOnly={isReadOnly}
+          required={isRequired}
+          type={inputTypeValue}
+          value={text('value', '')}
+        />
+        <FormHelperText hasError={hasError}>
+          {text('children', 'Info that helps a user with this field.')}
+        </FormHelperText>
+      </>
+    );
+  })
+  .add('Input', () => {
+    const uid = useUID();
+    return (
+      <>
+        <FormLabel for={uid}>Label Text</FormLabel>
+        <FormInput id={uid} type="text" placeholder="Placeholder" />
+        <FormHelperText>Info that helps a user with this field.</FormHelperText>
+      </>
+    );
+  })
+  .add('Input - Required', () => {
+    const uid = useUID();
+    return (
+      <>
+        <FormLabel for={uid} required>
+          Label
+        </FormLabel>
+        <FormInput id={uid} type="text" placeholder="Placeholder" required />
+        <FormHelperText>Info that helps a user with this field.</FormHelperText>
+      </>
+    );
+  })
+  .add('Input - Error', () => {
+    const uid = useUID();
+    return (
+      <>
+        <FormLabel for={uid}>Label</FormLabel>
+        <FormInput id={uid} type="text" placeholder="Placeholder" hasError />
+        <FormHelperText hasError>
+          <Flex>
+            <Box marginRight="space20">
+              <WarningIcon iconColor="colorTextError" decorative={false} title="warning: " size="sizeIcon30" />
```

Comment:
Should we handle this status level icon for the consumer? In the same way we handle it for alerts, the level should come with an icon for accessibility reasons, so we handle that for you. The content _next_ to it is up to you still.

https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/alert/src/index.tsx#L74

Reply:
I restructured this to include the warning icon.